order.js

```diff
@@ -2,7 +2,7 @@ API.load = (urlParams) => {
   Render.loading('main');
```

Comment:
oh, did i just review a useless document again?
The real changes are in order.js?

Delete order-dev and commit please. 🤦‍♀️ that just cost me like half an hour commenting on the wrong stuff.